INTERIOR/DisableMetroSystem.md

```diff
@@ -0,0 +1,35 @@
+---
+ns: INTERIOR
+aliases: ["_0x9E6542F0CE8E70A3"]
```

Comment:
Remove the underscore. It's generally not used when natives are given names.

Reply:
Underscores are only needed whenever converting names of a native, you don't need it for hashes because they're not included in the `Hash` codegen for C#

Example of when it would be needed would be if this was previously called `_DISABLE_METRO` and you renamed it to `DISABLE_METRO_SYSTEM` then you have to copy the native name with its underscore

Reply:
Please remove the underscore here.
```suggestion
aliases: ["0x9E6542F0CE8E70A3"]
```